### PR TITLE
Star Wars SWAPI (Independent Publisher)

### PR DESCRIPTION
**Pull request by paulculmsee**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 

image

- [x ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.

image

- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Thanks for the Star Wars connector submission! Hoping that people use this to learn how to use the Power Platform :) Given that this week is Thanksgiving week, we will pause certification processes and look at PRs starting Monday, 11/29. Hope you have a great week!

File: independent-publisher-connectors/Star Wars/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Paul Culmsee",
+	"stackOwner": "https://swapi.dev/"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Star Wars/apiProperties.json:7]: @paulculmsee Please remove the extra space.

File: independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2444 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenNEM",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json:4]: @paulculmsee We usually name the connector folders as per their title. Can you please rename the connector with its title `"OpenNEM"`.

File: independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2444 @@
...
+                              "type": "string",
+                              "description": "dispatch type"
+                            },
+                            "active": {
+                              "type": "boolean",
+                              "description": "active"
+                            },
+                            "capacity_registered": {
+                              "type": "number",
+                              "description": "capacity registered"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json:247]: @paulculmsee Few description values are Camel cased and a few are not in the swagger, Can you please follow a single pattern for all the description values of all the properties.

File: independent-publisher-connectors/Star Wars/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,878 @@
+{
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Star Wars/apiDefinition.swagger.json:1]: @paulculmsee Please raise one PR for each connector which makes us easy to review and approve it.

File: independent-publisher-connectors/Star Wars/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,878 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Star Wars",
+    "description": "A long time ago in a galaxy far, far away....",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Star Wars/apiDefinition.swagger.json:5]: @paulculmsee It looks good as a caption for your connector, but we need a description for your connector which can be understandable by everyone. Please provide a detailed description that should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Star Wars/apiDefinition.swagger.json (hunk 4 above)
Comment (paulculmsee) [reply independent-publisher-connectors/Star Wars/apiDefinition.swagger.json:1]: Sorry mate, did not realise I even did this :-(

File: independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json (hunk 3 above)
Comment (paulculmsee) [reply independent-publisher-connectors/CO2 Signal/apiDefinition.swagger.json]: Part of a problem on my end. Co2 Signal is another connector not meant to be part of this PR

File: independent-publisher-connectors/Star Wars/apiDefinition.swagger.json (hunk 5 above)
Comment (paulculmsee) [reply independent-publisher-connectors/Star Wars/apiDefinition.swagger.json:5]: That one I thought might get through on pop culture alone. But corrected with API description

File: independent-publisher-connectors/Star Wars/apiProperties.json (hunk 1 above)
Comment (sriyen-msft) [reply independent-publisher-connectors/Star Wars/apiProperties.json:7]: @paulculmsee Sorry for the delay in response, Can you please resolve this comment to get your PR approved and merged.

Comment (paulculmsee) [reply independent-publisher-connectors/Star Wars/apiProperties.json:7]: Sorry mate. Slowly getting better at this

File: independent-publisher-connectors/Star Wars/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,862 @@
+{
+	"swagger": "2.0",
+	"info": {
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Star Wars/apiDefinition.swagger.json:3]: @paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: independent-publisher-connectors/Star Wars/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,862 @@
...
+					"200": {
+						"description": "An array of species.",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"count": {
+									"type": "integer",
+									"format": "int32",
+									"description": "Total number of Species",
+									"title": "Species Count"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Star Wars/apiDefinition.swagger.json:35]: @paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Star Wars/apiDefinition.swagger.json (hunk 7 above)
Comment (paulculmsee) [reply independent-publisher-connectors/Star Wars/apiDefinition.swagger.json]: @sriyen-msft apologies (I am slowly getting the hang of it)

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@paulculmsee Apologies for the delay in response.
Update - Your connector is now in production deployment and reached [Test](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#region-deployment) region, and it would be completed and available in all the production regions within 3 weeks. Please let us know if you have any questions, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@paulculmsee Apologies for the delay in response.

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**